tests/AppBundle/Controller/EventControllerTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Tests\AppBundle\Controller;
+
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Tests\AppBundle\TestTools;
+
+class EventControllerTest extends WebTestCase
+{
+    public function testShowAllEvents()
+    {
+        TestTools::getLoggedInStavoAmbrone()->request('GET', '/events/show/all');
+
+        $this->assertContains('TestEvent1', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent2', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent3', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent4', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent5', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent6', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent7', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent8', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent9', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+        $this->assertContains('TestEvent10', TestTools::getLoggedInStavoAmbrone()->getResponse()->getContent());
+
+    }
+
+    public function testAddEvent()
```

Comment:
Test für den Fall, dass das Event schon exestiert

Reply:
Darf der Name denn nur einmal vorkommen? Was ist, wenn das gleiche Event in unterschiedlichen Jahren statt finden soll?

Reply:
Das heißt es Pfila 2017 Pfila 2016